content/en/blog/_posts/2024-02-22-ensure-secret-pulled-images.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: blog
+title: 'Image Pull Policy: Configuring Kubernetes to Ensure Access Security for Container Images in a Multi-Tenant Environment'
+date: 2024-02-22
+slug: ensure-secret-pulled-images
+---
+
+**Authors:** Michael Brown (IBM), Paco Xu (DaoCloud)
+
+Intro..
```

Comment:
This is where to talk about multitenancy use cases:
- SaaS multitenancy, where different tenants may not be running the exact same code
- isolating between teams in a common cluster (eg, cluster per environment; set-of-clusters per environment with tenants sharded over clusters)
- platform-as-a-service multitenancy

etc